cubes/new_cube.py

```diff
@@ -0,0 +1,254 @@
+""" JXP VERSION """
+
+'''
+Questions for Clancy:
+
+1. lC vs. C
```

Comment:
lC. For cubing, it has absolutely no preference at all, in the sense that the numerical minimiser searching for the optimum C can begin with a pretty good guess. However, when implementing some generic minimum fining algorithm, I expect it would be easier to operate on the log10. Also keep in mind that I am keeping with log10 for all logs here.